grafast/grafserv/__tests__/exampleServer.ts

```diff
@@ -4,7 +4,7 @@ import type { AddressInfo } from "node:net";
 import { constant, error, makeGrafastSchema } from "grafast";
 import { resolvePreset } from "graphile-config";
 
-import { grafserv } from "../src/servers/node/index.js";
+import { grafserv } from "../src/servers/whtatwg-node-server";
```

Comment:
Typo? Should be `whatwg`?

Also please don't update the example server; the current one does not add additional dependencies (just uses `node`) and thus is preferred.

Reply:
Correct, fixed